src/plugins/irc-events/ctcp.js

```diff
@@ -1,17 +1,17 @@
 "use strict";
 
-var Msg = require("../../models/msg");
+const Msg = require("../../models/msg");
 
-module.exports = function(irc, network) {
-	var client = this;
+module.exports = (irc, network) => {
```

Comment:
Yeah probably better to keep the function in exports. I'm not sure what scope it would get doing this.